src/components/common/TmSessionExplore.vue

```diff
@@ -58,9 +58,9 @@
           />
           <TmFormMsg
             v-else-if="
-              $v.address.$error && !$v.address.isNotAnyOtherCosmosAddress
+              $v.address.$error && !$v.address.isAWhitelistedBech32Prefix
             "
-            name="You can only sign in with a regular Cosmos address, starting with 'cosmos1'"
+            name="You can only sign in with a regular address"
```

Comment:
this is not a good error message. what does "regular" actually mean to a user?

Reply:
I know, but it is difficult to find a broad enough message to cover all networks... Any suggestions?

Reply:
> difficult to find a broad enough message to cover all networks

true! then the error messages should be network specific.

for now: "We don't currently support this type of address. Please try a different address format." ...?